terraform/environments/dev/terraform.tfvars

```
@@ -0,0 +1,30 @@
+# ------------------------------------------------------------------
+# IAM
+cloudwatch_lambda_logs_policy_name = "cloudwatch-lambda-logs-policy"
+# ------------------------------------------------------------------
+# S3 
+# uploads bucket
+s3_bucket_name          = "sdc-uploads"
+criminal_check_folder   = "criminal-check"
+income_proof_folder     = "income-proof"
+curriculum_plans_folder = "curriculum-plans"
+other_folder            = "other"
+# ------------------------------------------------------------------
+# Parameter Store (SSM)
+# zoom-class-link
+zoom_class_link_description = "Link used to access virtual classes hosted over Zoom"
+zoom_class_link_name        = "zoom-class-link"
+zoom_class_link_value       = "https://us06web.zoom.us/j/8391781258?pwd=NHBlMVh5SHc5UE10K2dsMFFvTzdmUT09"
+zoom_class_link_type        = "String"
+
+# ------------------------------------------------------------------
+# Lambda
+lambda_runtime = "python3.8"
+# cronMailing
+cronMailing_rule_name           = "hourly-cron-mailing"
+cronMailing_schedule_expression = "cron(0 * ? * * *)"
```

is this tested? isn't this supposed to be:
```suggestion
cronMailing_schedule_expression = "cron(0 * * * *)"
```

---

mm yeah I tested the expression and it showed me the correct times, but yeah I'll change it!. It was from an amazon doc on sagemaker which I don't think completely applies https://docs.aws.amazon.com/sagemaker/latest/dg/model-monitor-schedule-expression.html